database_schemas/DB_generation_schema.sql

```diff
@@ -1,131 +1,176 @@
 DROP TABLE IF EXISTS base_parameter;
 DROP TABLE IF EXISTS corner_plot;
 DROP TABLE IF EXISTS parameter_config;
-SET FOREIGN_KEY_CHECKS = 1;
+DROP TABLE IF EXISTS dataset_config;
+DROP TABLE IF EXISTS dataset_sigma;
+DROP TABLE IF EXISTS dataset_quantile;
 
--- Generate tables and set their primary keys as constraints
+SET foreign_key_checks = 1;
 
+-- Generate tables and set their primary keys as constraints
 CREATE TABLE rabit_user
-(
-    user_id      CHAR(36)   NOT NULL,
-    display_name VARCHAR(255) NOT NULL,
-    email        VARCHAR(255) NOT NULL
-);
+  (
+     user_id      CHAR(28) NOT NULL,
+     display_name VARCHAR(255) NOT NULL,
+     email        VARCHAR(255) NOT NULL
+  );
 
 ALTER TABLE rabit_user
-    ADD CONSTRAINT rabit_user_pk PRIMARY KEY (user_id);
+  ADD CONSTRAINT rabit_user_pk PRIMARY KEY (user_id);
 
 CREATE TABLE upload
-(
-    upload_id       CHAR(36) NOT NULL,
-    user_id         CHAR(36) NOT NULL,
-    upload_datetime DATETIME
-);
+  (
+     upload_id       CHAR(36) NOT NULL,
+     user_id         CHAR(36) NOT NULL,
+     upload_datetime DATETIME
+  );
 
 ALTER TABLE upload
-    ADD CONSTRAINT upload_pk PRIMARY KEY (upload_id);
+  ADD CONSTRAINT upload_pk PRIMARY KEY (upload_id);
 
 CREATE TABLE file_pointer
-(
-    upload_id     CHAR(36)   NOT NULL,
-    collection_id CHAR(36)   NOT NULL
-);
+  (
+     file_id       CHAR(36) NOT NULL,
+     upload_id     CHAR(36) NOT NULL,
+     collection_id CHAR(36) NOT NULL
+  );
 
 ALTER TABLE file_pointer
-    ADD CONSTRAINT file_pointer_pk PRIMARY KEY (upload_id, collection_id);
+  ADD CONSTRAINT file_pointer_pk PRIMARY KEY (file_id);
 
 CREATE TABLE plot_collection
-(
-    collection_id CHAR(36) NOT NULL,
-    collection_name VARCHAR(255) NOT NULL
-);
+  (
+     collection_id          CHAR(36) NOT NULL,
+     collection_title       VARCHAR(255) NOT NULL,
+     collection_description VARCHAR(1023)
+  );
 
 ALTER TABLE plot_collection
-    ADD CONSTRAINT plot_collection_pk PRIMARY KEY (collection_id);
+  ADD CONSTRAINT plot_collection_pk PRIMARY KEY (collection_id);
 
 CREATE TABLE base_parameter
-(
-    parameter_id   CHAR(36)  NOT NULL,
-    parameter_name VARCHAR(2) NOT NULL,
-    collection_id  CHAR(36)  NOT NULL
-);
+  (
+     parameter_id   CHAR(36) NOT NULL,
+     parameter_name VARCHAR(2) NOT NULL,
+     file_id        CHAR(36) NOT NULL
+  );
 
 ALTER TABLE base_parameter
-    ADD CONSTRAINT base_parameter_pk PRIMARY KEY (parameter_id);
+  ADD CONSTRAINT base_parameter_pk PRIMARY KEY (parameter_id),
+  ADD CONSTRAINT base_parameter_unique UNIQUE (parameter_name, file_id);
 
 CREATE TABLE corner_plot
-(
-    corner_id     CHAR(36) NOT NULL,
-    plot_locked   BOOLEAN    NOT NULL,
-    collection_id CHAR(36) NOT NULL,
-    user_id       CHAR(36) NOT NULL
-);
+  (
+     corner_id         CHAR(36) NOT NULL,
+     last_modified     DATETIME,
+     date_created      DATETIME,
+     collection_id     CHAR(36) NOT NULL,
+     user_id           CHAR(36) NOT NULL,
```

Comment:
```suggestion
     user_id           CHAR(28) NOT NULL,
```
`user_id` should have type `CHAR(28)`